versioned_plugins.rb

```diff
@@ -33,6 +33,10 @@ class VersionedPluginDocs < Clamp::Command
     "logstash-codec-java_codec_example"
   ]
 
+  ADDITIONAL_PLUGINS = [
+    "elastic/logstash-filter-elastic_integration"
```

Comment:
review note: `logstash-plugins` org is on our hand. We automatically observe plugins under it and list up the plugins to skip. However, `elastic` org is out of our control and plugins can be created anytime even if we list up skip list. So, for the simplicity, I have hardcoded for now and open for any better methods.

Reply:
let's add that explanation to the constant